src/View/Helper/FlashMessenger.php

```diff
@@ -89,6 +89,25 @@ class FlashMessenger extends AbstractHelper
      */
     protected $pluginFlashMessenger;
 
+    /**
+     * All namespaces of FlashMessenger
+     *
+     * Keys of array are the namespace names
+     *
+     * @var array<string, FlashMessengerNamespace> $namespaces
+     */
+    private $namespaces = [];
+
+    /**
+     * @param array<string, FlashMessengerNamespace> $namespaces
+     */
+    public function __construct(array $namespaces = [])
+    {
+        foreach ($namespaces as $namespace) {
+            $this->namespaces[$namespace->getName()] = $namespace;
+        }
+    }
```

Comment:
The input is never validated that means I can write:

```php
$flashMessenger = new FlashMessenger([new Laminas\Form\Element\Text('name')]);
```

…and everything is fine so far.

Reply:
If that is enough, then okay.